.github/workflows/interface-dependency-versions.yml

```diff
@@ -63,11 +63,11 @@ jobs:
       
       - name: Nightly Catalyst Version
         id: catalyst
-        run: echo "nightly=--extra-index-url https://test.pypi.org/simple/ PennyLane-Catalyst --pre" >> $GITHUB_OUTPUT
+        run: echo "nightly=--index https://test.pypi.org/simple/ --prerelease=allow --upgrade-package PennyLane-Catalyst PennyLane-Catalyst" >> $GITHUB_OUTPUT
       
       - name: PennyLane-Lightning Latest Version
         id: pennylane-lightning
-        run: echo "latest=-i https://test.pypi.org/simple/ PennyLane-Lightning --pre --upgrade" >> $GITHUB_OUTPUT
+        run: echo "latest=--default-index https://test.pypi.org/simple/ --prerelease=allow --upgrade-package PennyLane-Lightning PennyLane-Lightning" >> $GITHUB_OUTPUT
```

Comment:
Same q here as above

Reply:
This would indeed replace the pypi.org target

Reply:
Do we want to do this? Since some deps may not be available on test pypi?

Reply:
Unless there's good reason I'm unaware of, I'd say the original `-i` flag can defer to the extra index equialent.

Reply:
Because the only dependency of lightning (listed) is pennylane itself, overriding the default didn't cause any issues. But I can indeed swap it to `--index`, do not think we will hit any problems.

Reply:
If we need bumped versions in Lightning I think we will hit issues, or if we require additional packages that arent in PL (such as if adding LG/LK/LT)

But if there's no issue, I'm fine with it for now.

Reply:
I've updated this to `--index`